spec/factories/week_predictions.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :week_prediction do
+    fantasy_score "9.99"
+    player nil
+    season_week nil
```

Comment:
I think we should make player and season_week factories instead of tying this to nil